### PR TITLE
Refactor assets and provide new theme

### PR DESCRIPTION
**Pull request by sentrychris**
Just some general refactoring:

- Placed styles in their own external stylesheets and script in app.js
- Created new directory `resources/` containing `assets/` and `templates/`
- Created an asset helper function
- Created a new theme

Let me know if you'd like me to change anything :)

### CODE REVIEW

File: resources/templates/darcula/terminal.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?php
+
+/** @var $this \CaptainSpain\WebTerminal\Terminal */
+
+function makeInputLine($data, $fromData = true)
```
Comment (captainspain) [inline resources/templates/darcula/terminal.php]: This function is exactly the same as in `templates/default.php`, can you move this function to `helpers.php` or create a seperate file for functions that are being used in templates.

**Review by captainspain — CHANGES_REQUESTED** [review]
Thanks a lot!
1 minor issue, if you can fix that one I will merge it.

File: resources/templates/darcula/terminal.php (hunk 1 above)
Comment (sentrychris) [reply resources/templates/darcula/terminal.php]: No problem! I will update the PR shortly.

**Review by captainspain — APPROVED** [review]

**Comment by captainspain** [comment]
Thank you!

**Pull request merged**